src/e2e-test/features/salesforcemultiobjectsbatchsource/RunTime.feature

```diff
@@ -131,4 +131,46 @@ Feature: Salesforce Multi Objects Batch Source - Run time Scenarios
     And Wait till pipeline is in running state
     And Open and capture logs
     And Verify the pipeline status is "Succeeded"
-    Then Close the pipeline logs
\ No newline at end of file
+    Then Close the pipeline logs
+
+
+
+  @MULTIBATCH-TS-SF-RNTM-04 @BQ_SINK_TEST @BQ_TEMP_CLEANUP @CREATE_CUSTOM @DELETE_OBJECT
```

Comment:
Same thing here update the above tests with data validation logic no need to add a new test, also the object are same.

Reply:
done